datatypes/src/main/java/org/hyperledger/besu/datatypes/AccessListEntry.java

```diff
@@ -0,0 +1,22 @@
+package org.hyperledger.besu.datatypes;
+
+import java.util.List;
+
+import org.apache.tuweni.bytes.Bytes32;
+
+/** An access list entry as defined in EIP-2930 */
+public interface AccessListEntry {
```

Comment:
Can this be a record instead of an interface?

Even if it's not converted to a record, I'd rather see the whole class pulled in from the evm module to the datatypes module instead of an interface with a single implementation.

Reply:
As suggested I've removed the interface and created a record instead.